tests/app/clients/test_zendesk_sell.py

```diff
@@ -293,26 +296,27 @@ def match_json(request):
             assert not deal_id
 
 
-def test_send_create_service_contact_fail(
+def test_common_create_or_go_live_contact_fail(
```

Comment:
WYDT about not testing `_common_create_or_go_live` directly but rather testing fail cases for create service + go live service? You can `pytest.parametrize` those methods to make sure that they work as intended without duplicated test cases.